CMakeLists.txt

```diff
@@ -29,10 +29,17 @@
 #**                                                                 **
 #*********************************************************************
 
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8.2)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.0)
```

Comment:
Didn't understand, why this version bump is needed. Is cmake 3 available in Trusty?

Reply:
Have a look at [CMP0048](https://cmake.org/cmake/help/v3.0/policy/CMP0048.html). Since CMake 2.8 does not support `Version` in `PROJECT()` the version had to be pushed. The [ci-jobs](https://travis-ci.org/iclcv/icl/jobs/283380218) use Ubuntu 14.04 LTS with CMake 3.2.2 but the (docker) images seem to be altered as Ubuntu 14.04 officially only support CMake 2.8.2. I will change the required version back to 2.8.2 and change the related policy with the help of `IF(CMAKE_VERSION VERSION_LESS ...)`